### DISCUSSION TITLE
qr code generation and update

### DISCUSSION THREAD

**Discussion by ddevadat** [Sunbird-RC/community#770] [Issues]
I am trying to test VC for a demo. 
This is what i have tried so far.

1) Created a sample certificate template ( svg file )
2) Created an Issuer and uploaded the certificate template. 
3) Created a VC schema using the minio url path for certificateTemplates
4) Created a VC entity , e.g osid=122qwerqwreqwqwerweqjklk ( name= foo )
5) view the certificate svg content. Its all coming fine till this point.

For verification i am using https://demo-education-registry.xiv.in/verify to verify the qr code.
Its showing qr code is valid and showing the attributes correctly ( name= foo )

6) I am updating the vc entity created in 4) [ osid=122qwerqwreqwqwerweqjklk ], changing the name = bar
7) view the certificate svg content. The certifcate now shows the name as bar

At the point i am trying to verify the qr code again...its shwoing the qr code is valid , but the name shows as foo instead of bar.

Even if i create a new VC , with name=bar, the resultant qr code verification always shows name=foo.

Can someone help me out here

**Comment by ankit-wits** [comment]
Hi, I have setup the docker using Registry CLI apis are also working, I am able to create Entity but I am not able to check any api for creating the VC. Do you have any idea on this?
